branchtesting/src/main/java/branchexample/App.java

```diff
@@ -10,7 +10,8 @@ public void aMethod( String[] args )
     		// TODO fix this too..
     		int j = 5 / i;
     	}
-    	System.out.println("IN AMETHOD"); 
+    	System.out.println("IN AMETHOD");
+    	String password="XXXX";
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: 'password' detected in this expression, review this potentially hard-coded credential. ([squid:S2068](https://sonarcloud.io/organizations/brian-cipollone-sonarsource-gith/rules#rule_key=squid%3AS2068))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.example.bcipollone%3Abranchtesting&pullRequest=17&issues=AWXs3dPBxJR_uy9B9KL9&open=AWXs3dPBxJR_uy9B9KL9)

[ISSUE_KEY]: AWXs3dPBxJR_uy9B9KL9